Formula/chruby.rb

```diff
@@ -17,31 +17,4 @@ class Chruby < Formula
   def install
     system "make", "install", "PREFIX=#{prefix}"
   end
-
-  def caveats; <<-EOS.undent
-    Add the following to the ~/.bashrc or ~/.zshrc file:
-
-      source #{opt_share}/chruby/chruby.sh
```

Comment:
Let's trim the caveats down to just this line and the lines above and skip the empty line.